components/header/index.tsx

```diff
@@ -30,28 +31,74 @@ export default function Header({
           <div className="-mr-2 -my-2 md:hidden">
             <button
               type="button"
-              className="bg-white rounded-md p-2 inline-flex items-center justify-center text-gray-400 hover:text-gray-500 hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-inset focus:ring-gray-500"
+              className="bg-black rounded-md p-2 inline-flex items-center justify-center text-gray-400 hover:text-gray-500 hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-inset focus:ring-gray-500"
               aria-expanded="false"
               onClick={() => onMobileMenuCollapsedChange(true)}>
               <span className="sr-only">Open menu</span>
               <FontAwesomeIcon icon={faBars} />
             </button>
           </div>
           <div className="hidden md:flex items-center justify-end md:flex-1 lg:w-0">
-            <a
-              href="https://hyperjumptech.github.io/monika/overview"
-              target="_blank"
-              rel="noopener noreferrer"
-              className="whitespace-nowrap text-base font-medium text-gray-400 hover:text-gray-900">
-              Documentation
-            </a>
-            <a
-              href="https://hyperjumptech.github.io/monika"
-              target="_blank"
-              rel="noopener noreferrer"
-              className="ml-8 whitespace-nowrap inline-flex items-center justify-center px-4 py-2 border border-transparent rounded-md shadow-sm text-base font-medium text-white bg-gray-700 hover:bg-gray-800">
-              Get Monika
-            </a>
+            <ul className="flex flex-row list-none lg:ml-auto">
+              <li className="nav-item">
+                <a
+                  className="px-3 py-2 flex items-center text-white font-sans font-bold leading-snug hover:opacity-75"
+                  href="https://monika.hyperjump.tech/overview">
+                  <span className="ml-2">Docs</span>
+                </a>
+              </li>
+              <li className="nav-item">
+                <span className="ml-2 px-3 py-2 flex items-center text-white font-sans underline font-bold leading-snug hover:opacity-75">
```

Comment:
![image](https://user-images.githubusercontent.com/16670475/128169961-45e4f72e-bc2c-4825-b266-e09b61850858.png)
Even when this is currently active, I think it's better to be clickable and redirect you to the monika config generator index page